docs/views-and-variables.rst

```diff
@@ -61,7 +61,53 @@ say
 While the ``tell`` and ``ask`` view types are an exact representation of the base alexa programming model, the say views are different. They're an abstraction created by voxa to make it simpler to compose your responses over many state transitions. They don't send a respond to alexa but instead make a state transition internally and continue executing your skill code until there's a ``tell`` or ``ask`` response.
 
 
+Directives
+-----------
+
+You can include directives in your views file. This will allow you to set directive for Display.RenderTemplate
```

Comment:
Grammer: This sentence isn't saying anything to me, "This will allow you to set directive for Display.RenderTemplate"